src/react/components/common/cloudFilePicker/cloudFilePicker.tsx

```diff
@@ -145,6 +145,7 @@ export class CloudFilePicker extends React.Component<ICloudFilePickerProps, IClo
             items={items}
             Component={ListItem}
             onClick={onClick}
+            displayEmptyMessage={true}
```

Comment:
Let's set default values on this prop so we don't have to explicitly go back and make changes in all areas.